multiqc/modules/whatshap/whatshap.py

```diff
@@ -95,7 +95,7 @@ def process_data(data):
             data = {field: value for field, value in zip(header, spline)}
             process_data(data)
             # Remove the sample and chromsome from the data
-            sample = self.clean_s_name(data.pop("sample"), logfile)
+            data.pop("sample")
```

Comment:
Any reason to revert this change? I think I added the `self.clean_s_name()` function here because otherwise it's not used for the plot and sample renaming / `-d` `-s` etc flags don't work here..

Reply:
The problem was that the sample name was getting cleaned twice, once inside the loop and once outside. I've now move the cleaning to just before we return the sample name.

Reply:
Ah sorry, that was my mistake when I added this then 👍🏻